### PR TITLE
Fix/issue 1167 hide add selected button on empty storage

### PR DESCRIPTION
**Pull request by santiagonoguez**
## Description
Hide 'add selected' button when there are no files that can be selected in a folder.

## Motivation and Context
See issue 1167 https://github.com/Rise-Vision/rise-vision-apps/issues/1167

## How Has This Been Tested?
See https://github.com/Rise-Vision/common-header/pull/1038
for this part and the release checklist.

### CODE REVIEW

**Comment by santiagonoguez** [comment]
@willread updated to latest master ! :D

File: web/scripts/template-editor/services/svc-template-editor-utils.js [hunk 1]
```diff
@@ -64,6 +64,14 @@ angular.module('risevision.template-editor.services')
         });
       };
 
+      svc.hasRegularFileItems = function (folderItems) {
```
Comment (stulees) [inline web/scripts/template-editor/services/svc-template-editor-utils.js]: @santiagonoguez This is minor, but just wondering about this function name, why _Regular_ is in the name? Is this to differentiate it from something else?

Comment (santiagonoguez) [reply web/scripts/template-editor/services/svc-template-editor-utils.js]: @stulees yes, it's to differentiate regular files from folders ( selectedFiles in contrast contains both folders and regular files ). As with HTML templates we still don't support folder selection, showing the 'add selected' button when there are only folders and no regular files makes no sense. Thanks !

Comment (stulees) [reply web/scripts/template-editor/services/svc-template-editor-utils.js]: @santiagonoguez Okay thanks for clarifying

**Pull request merged**